judge/migrations/0116_convert_to_dates.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 2.2.17 on 2021-04-27 14:46
+
+from django.db import migrations, models
+from django.db.models import F
+from django.utils import timezone
+
+
+def convert_to_datetime(apps, schema_editor):
+    Contest = apps.get_model('judge', 'Contest')
+    Submission = apps.get_model('judge', 'Submission')
+
+    Submission.objects.filter(was_rejudged=True).update(rejudged_date=F('judged_date'))
+    Submission.objects.filter(is_locked=True).update(locked_after=timezone.now())
+    Contest.objects.filter(is_locked=True).update(locked_after=F('end_time'))
+
+
+def convert_to_boolean(apps, schema_editor):
+    Contest = apps.get_model('judge', 'Contest')
+    Submission = apps.get_model('judge', 'Submission')
+
+    Submission.objects.filter(rejudged_date__isnull=False).update(was_rejudged=True)
+    Submission.objects.filter(locked_after__isnull=False).update(is_locked=True)
+    Contest.objects.filter(locked_after__isnull=False).update(is_locked=True)
```

Comment:
nit: I would only set this if the date is after `timezone.now()`, although in practice I highly doubt this matters

Reply:
Updated.